corehq/apps/importer/templates/importer/partials/import_status.html

```diff
@@ -80,8 +80,12 @@
         </p>
     {% endif %}
     {% if progress.percent %}
-        <div class="progress progress-striped active">
-            <div class="bar" style="width: {{ progress.percent }}%;"></div>
+        <div class="progress">
+            <div class="progress-bar progress-bar-info"
+                 role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100"
```

Comment:
should `aria-valuenow` be `{{progress.percent}}`? 
Didn't realize we did accessibility stuff... if this isn't really something we are supporting, might be worth just removing the `aria` values all together?